.craft.yml

```diff
@@ -6,101 +6,104 @@ targets:
-    id: "@sentry/browser"
+    id: '@sentry/browser'
     includeNames: /^sentry-browser-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/node"
+    id: '@sentry/node'
     includeNames: /^sentry-node-\d.*\.tgz$/
 
   ## 3 Browser-based Packages
   - name: npm
-    id: "@sentry/angular-ivy"
+    id: '@sentry/angular-ivy'
     includeNames: /^sentry-angular-ivy-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/angular"
+    id: '@sentry/angular'
     includeNames: /^sentry-angular-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/ember"
+    id: '@sentry/ember'
     includeNames: /^sentry-ember-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/react"
+    id: '@sentry/react'
     includeNames: /^sentry-react-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/svelte"
+    id: '@sentry/svelte'
     includeNames: /^sentry-svelte-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/vue"
+    id: '@sentry/vue'
     includeNames: /^sentry-vue-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/wasm"
+    id: '@sentry/wasm'
     includeNames: /^sentry-wasm-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/integrations"
+    id: '@sentry/integrations'
     includeNames: /^sentry-integrations-\d.*\.tgz$/
 
   ## 4. Node-based Packages
   - name: npm
-    id: "@sentry/serverless"
+    id: '@sentry/serverless'
     includeNames: /^sentry-serverless-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/opentelemetry-node"
+    id: '@sentry/opentelemetry-node'
     includeNames: /^sentry-opentelemetry-node-\d.*\.tgz$/
 
   ## 5. Fullstack/Meta Frameworks (depending on Node and Browser or Framework SDKs)
   - name: npm
-    id: "@sentry/nextjs"
+    id: '@sentry/nextjs'
     includeNames: /^sentry-nextjs-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/remix"
+    id: '@sentry/remix'
     includeNames: /^sentry-remix-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/sveltekit"
+    id: '@sentry/sveltekit'
     includeNames: /^sentry-sveltekit-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/gatsby"
+    id: '@sentry/gatsby'
     includeNames: /^sentry-gatsby-\d.*\.tgz$/
 
   ## 6. Other Packages
   ## 6.1
   - name: npm
-    id: "@sentry-internal/typescript"
+    id: '@sentry-internal/typescript'
     includeNames: /^sentry-internal-typescript-\d.*\.tgz$/
   - name: npm
-    id: "@sentry-internal/eslint-plugin-sdk"
+    id: '@sentry-internal/eslint-plugin-sdk'
     includeNames: /^sentry-internal-eslint-plugin-sdk-\d.*\.tgz$/
   ## 6.2
   - name: npm
-    id: "@sentry-internal/eslint-config-sdk"
+    id: '@sentry-internal/eslint-config-sdk'
     includeNames: /^sentry-internal-eslint-config-sdk-\d.*\.tgz$/
 
   ## 7. Deprecated packages we still release (but no packages depend on them anymore)
   - name: npm
-    id: "@sentry/hub"
+    id: '@sentry/hub'
     includeNames: /^sentry-hub-\d.*\.tgz$/
   - name: npm
-    id: "@sentry/tracing"
+    id: '@sentry/tracing'
     includeNames: /^sentry-tracing-\d.*\.tgz$/
+  - name: npm
+    id: '@sentry/node-experimental'
+    includeNames: /^sentry-node-experimental-\d.*\.tgz$/
```

Comment:
l: let's maybe not put this under 7 because it's not deprecated, wdyt?
```suggestion

## 8. Experimental packages

  - name: npm
    id: '@sentry/node-experimental'
    includeNames: /^sentry-node-experimental-\d.*\.tgz$/
```

Reply:
ah damn, I didn't read this well enough xD totally!